CHANGELOG.md

```diff
@@ -1,9 +1,3 @@
-# HEAD
```

Comment:
Looks like this got removed by mistake in a rebase. We should also add this change to the changelog

Reply:
Wow, it managed to remove bits of both the PRs involved in the rebase.